web/src/components/filings-calendar/FilingsCalendar.test.tsx

```diff
@@ -131,12 +141,12 @@ describe("<FilingsCalendar />", () => {
   });
 
   it("displays filings only within a year in list view", () => {
-    const farDueDate = getCurrentDate().add(2, "years");
+    const farDueDate = getAprilDateOfThisYear().add(2, "years");
     const farReport = generateTaxFilingCalendarEvent({
       identifier: "whatever",
       dueDate: farDueDate.format(defaultDateFormat),
     });
-    const recentDueDate = getCurrentDate().add(2, "months");
+    const recentDueDate = getAprilDateOfThisYear().add(2, "months");
```

Comment:
[dust] As I understand it, we added two months to get some amount of time in the future, but now that we're specifically setting it to April, any reason we're still adding two months? Just curious. Perhaps a refactor to consider for another day.

Reply:
No idea, wasn't something we wanted to investigate given the blocking nature. Totally agree it's a good candidate for understanding and potential refactor